src/templates/interface/video/dynamic-reveal.js

```diff
@@ -0,0 +1,34 @@
+import { slotService, slotTweaker } from '@wikia/ad-engine';
+import { DEFAULT_VIDEO_ASPECT_RATIO } from '../../outstream/porvata-template';
+
+function add(video, container, params) {
+	const slot = slotService.get(params.slotName);
+
+	let slotExpanded = false;
+
+	video.addEventListener('loaded', () => {
+		if (!slotExpanded) {
+			slotTweaker.expand(slot);
+			slotExpanded = true;
+
+			// Delay dispatching event so it's run after browser really finish expanding the slot
+			setTimeout(() => {
+				video.ima.dispatchEvent('wikiaSlotExpanded');
+			}, 1000);
```

Comment:
Please describe why 1000ms is enough.

Reply:
It's related to animation time:
https://github.com/Wikia/ad-products/pull/132/files#diff-b4c62f19b8bcf3f02c9d6adbd9b7fa94R48
I'll add a comment here